lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -86,88 +84,37 @@ def paintEvent(self, e):
         if not hasattr(self, 'renderer'):
             return
 
-        if DEBUG:
-            print('FigureCanvasQtAgg.paintEvent: ', self,
-                  self.get_width_height())
-
-        if len(self.blitbox) == 0:
-            # matplotlib is in rgba byte order.  QImage wants to put the bytes
-            # into argb format and is in a 4 byte unsigned int.  Little endian
-            # system is LSB first and expects the bytes in reverse order
-            # (bgra).
-            if QtCore.QSysInfo.ByteOrder == QtCore.QSysInfo.LittleEndian:
```

Comment:
I'm just reusing the blitting codepath for both cases, so either `region.to_string_argb` (which is a different method from `renderer.tostring_argb`...) was already doing the right thing wrt. endianness, or there has been a bug for a while on one of the two endiannesses that no one ever noticed.